src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt

```diff
@@ -531,9 +535,13 @@ internal class ProjectPlugin(private val project: Project) {
     val computeDominatorTask = tasks.register<ComputeDominatorTreeTask>("computeDominatorTree$taskNameSuffix") {
       projectPath.set(thisProjectPath)
       physicalArtifacts.set(artifactsReportTask.flatMap { it.output })
+      runtimePhysicalArtifacts.set(artifactsReportTask.flatMap { it.outputRuntime })
       graphView.set(graphViewTask.flatMap { it.output })
-      outputTxt.set(outputPaths.dominatorConsolePath)
-      outputDot.set(outputPaths.dominatorGraphPath)
+      runtimeGraphView.set(graphViewTask.flatMap { it.outputRuntime })
+      outputTxt.set(outputPaths.compileDominatorConsolePath)
+      runtimeOutputTxt.set(outputPaths.runtimeDominatorConsolePath)
+      outputDot.set(outputPaths.compileDominatorGraphPath)
+      runtimeOutputDot.set(outputPaths.runtimeDominatorGraphPath)
     }
```

Comment:
Same suggestion as [above](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/pull/1042/files#r1400248269), perhaps this could be simpler to register seperate tasks for `computeCompileDominatorTree$taskNameSuffix` and `computeRuntimeDominatorTree$taskNameSuffix` that use the output from their respective `(compile|runtime)ArtifactsReportTask` tasks, then have a parent task for `computeDominatorTree$taskNameSuffix` that runs both.

Reply:
There was precedence for in the existing code for supporting both compile and runtime classpath in a single tasks (e.g. [ here](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/blob/a22aa2fa499fcb814b2295ee0d6a08efffeebca5/src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt#L497-L498)), so I opted to do something similar.

Reply:
That's fair. I'm not very familiar with this codebase either so please don't consider my suggestion as requirements. Perhaps @autonomousapps has an opinion on this?

Reply:
I could go either way. This particular task is not part of the primary analysis of this plugin. I basically added it because it was convenient to reuse all the machinery around graph analysis I already had set up here, and I thought maybe I'd find a good use for it eventually. At this point, _you_ are the main users of this, as far as I know. So as the users, you should tell me what you'd find most convenient / useful, and that works for me. What you have here, or separate tasks.